temp.c

```diff
@@ -173,65 +259,22 @@ void temp_sensor_tick() {
 
 				#ifdef	TEMP_THERMISTOR
 				case TT_THERMISTOR:
-					do {
-						uint8_t j, table_num;
-						//Read current temperature
-						temp = analog_read(i);
```

Comment:
Analog read is index based. So we don't read the temp_pin. We read the ADC of index i, which is configured in analog_xxx.c.